tests/integration/providers/ldap/delta_outbound_sync_tests.py

```diff
@@ -39,29 +41,23 @@
         {
             "data": {
                 "description": "The role that keeps on rolling",
-                "group_types": -2147483646,
-                "name": "Rolling_Role",
                 "members": [],
-                "owners": "CN=christopher,OU=Users,OU=Accounts,DC=AD2012,DC=LAB",
                 "remote_id": "CN=Rolling_Role,OU=Roles,OU=Security,OU=Groups,DC=AD2012,DC=LAB",
             },
             "data_type": "group",
-            "timestamp": r.epoch_time(1376074395.012).to_iso8601(),
+            "timestamp": r.epoch_time(1376074395.012),
             "provider_id": "test_provider",
             "status": "UNCONFIRMED",
             "action": "",
         },
         {
             "data": {
                 "description": "The role that keeps on rolling",
-                "group_types": -2147483646,
-                "name": "Rolling_Role",
                 "members": [],
-                "owners": "CN=christopher,OU=Users,OU=Accounts,DC=AD2012,DC=LAB",
                 "remote_id": "CN=Rolling_Role,OU=Roles,OU=Security,OU=Groups,DC=AD2012,DC=LAB",
             },
             "data_type": "group",
-            "timestamp": "2013-08-09T18:53:15.012+00:00",
+            "timestamp": dt.fromtimestamp(1376074395.012, tz=pytz.utc),
```

Comment:
why do the other two test entries use `r.epoch_time(1376074395.012)`, is this difference on purpose? If it is can you add a comment to explain the difference and why it's required?

Reply:
RethinkDB requires the r.epoch_time() to properly save the epoch time as a datetime object in RethinkDB. Here I created the datetime object manually from Python so I didn't need to use r.epoch_time()